nucypher/crypto/powers.py

```diff
@@ -97,22 +101,22 @@ def sign_message(self, message: bytes):
             raise PowerUpError("Account is not unlocked.")
 
         signature = self.blockchain.interface.call_backend_sign(self.account, message)
-        return signature
+        return signature.to_bytes()
 
-    def verify_message(self, address: str, pubkey: bytes, message: bytes, signature: str):
+    def verify_message(self, address: str, pubkey: bytes, message: bytes, signature_bytes: bytes):
```

Comment:
Not that it matters much, but is it necessary to have the type indicated in the argument name as `signature_bytes` rather than just naming `signature`?

Reply:
Yeah, this is a perennial wonder of mine.  Since we make a `Signature` object in the next line, doesn't it make sense to clarify that this object is a bytes representation?  I dunno.

Reply:
I'm not too strong on one or the other way though. It's not super-important here. So, I'm ok with `_bytes`